src/styles/main.scss

```
@@ -1,3 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Arial, sans-serif;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+// variables:
+$grey: #eee;
+$cell-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$cells-per-line: 7;
+$start-day:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  list-style-type: none;
+  counter-reset: my-counter;
+  width: $cell-size * $cells-per-line + $padding * 2 + ($cells-per-line * $gap);
+  &__cell {
+    width: $cell-size;
+    height: $cell-size;
+    background-color: $grey;
+    font-size: 30px;
+    position: relative; 
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #000;
+    transition: .3s ease;
+    &:before {
+      content: counter(my-counter);
+      counter-increment: my-counter;
+    }
+    &:hover{
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+  }
+
+  @each $day, $num in $start-day {
+    &--#{$day} {
+      & .calendar__cell:first-child{
```

```suggestion
      .calendar__cell:first-child{
```
Нужен ли тут амперсанд?